.agola/config.jsonnet

```diff
@@ -107,7 +107,7 @@ local task_build_push_images(name, target, push) =
         [
           task_build_go(version, arch),
         ]
-        for version in ['1.15', '1.16']
+        for version in ['1.17']
```

Comment:
go 1.18 has already been released.

Reply:
ok @sgotti I change the PR to update go 1.18